packages/appstore/src/components/wallet-header/__tests__/wallet-header.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+// import { isMobile } from '@deriv/shared';
+import { useStore } from '@deriv/stores';
+import WalletHeader from '..';
+
+const mocked_root_store: DeepPartial<ReturnType<typeof useStore>> = {
+    ui: {
+        is_dark_mode_on: false,
+    },
+};
+
+jest.mock('@deriv/stores', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    observer: <T,>(Component: T) => Component,
+    useStore: () => mocked_root_store,
+}));
```

Comment:
@sergei-deriv Please use `mockStore` instead 🙏🏻 You can search in the codebase for `mockStore` to see the usage in other test files 🙇🏻